Cargo.toml

```diff
@@ -1,3 +1,10 @@
+[workspace.package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+repository = "https://github.com/paritytech/polkadot-sdk.git"
+license = "GPL-3.0-only"
+version = "1.0.0"
```

Comment:
Also not sure about this. We have a lot of crates in Substrate that don't use this version as they are already on a much higher version.

Reply:
I only see crates in Polkadot use the `version.workspace = true` syntax. `rococo-runtime` for example.  
Dont know what to do with these. Maybe setting them manually to `0.0.1`? @Morganamilo would then notice if we already released higher versions of those i guess.

Reply:
Yeah I know that we used this in Polkadot, but now we can not really do this anymore.

Reply:
Okay I set it to `1.0.0` manually and removed the `version.workspace`. Now also added `edition`, `authors` and `repository`.